test/cli/package-error-missing-export/test.out

```diff
@@ -1,52 +1,32 @@
-foo_class.rb:10: Package `Foo::Bar::OtherPackage` does not export `Foo::Bar::OtherPackage::NotExported` https://srb.help/5002
+foo_class.rb:10: Package `Foo::Bar::OtherPackage` does not export `Foo::Bar::OtherPackage::NotExported` https://srb.help/3717
     10 |      Foo::Bar::OtherPackage::NotExported
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   Autocorrect: Use `-a` to autocorrect
     other/__package.rb:4: Insert `export Foo::Bar::OtherPackage::NotExported`
      4 |  export Foo::Bar::OtherPackage::OtherClass
                                                    ^
-    other/other_class.rb:7: Defined here
-     7 |  class NotExported
-          ^^^^^^^^^^^^^^^^^
-  Note:
-    Try running generate-packages.sh
 
-foo_class.rb:11: Package `Foo::Bar::OtherPackage` does not export `Bar::OtherPackage::NotExported` https://srb.help/5002
+foo_class.rb:11: Package `Foo::Bar::OtherPackage` does not export `Foo::Bar::OtherPackage::NotExported` https://srb.help/3717
     11 |      Bar::OtherPackage::NotExported
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   Autocorrect: Use `-a` to autocorrect
     other/__package.rb:4: Insert `export Foo::Bar::OtherPackage::NotExported`
      4 |  export Foo::Bar::OtherPackage::OtherClass
                                                    ^
-    other/other_class.rb:7: Defined here
-     7 |  class NotExported
-          ^^^^^^^^^^^^^^^^^
-  Note:
-    Try running generate-packages.sh
```

Comment:
Why did we drop the `generate-packages.sh` note?

Reply:
We had a lot of additional suggestions in SuggestPackage.cc that get conditionally applied when an error is raised from the resolver. Longer-term, migrating a lot of those into the visibility checker would be the best way to recover their functionality, but in the short term I can add this one back in.